### PR TITLE
Bump the cloud-sql-proxy to v1191

### PR DESCRIPTION
**Pull request by franpog859**
**Description**

Changes proposed in this pull request:

- Bump the cloud-sql-proxy to v1.19.1

### CODE REVIEW

**Comment by franpog859** [comment]
Do not merge this PR until everything is properly tested!

File: resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml [hunk 1]
```diff
@@ -98,7 +98,7 @@ spec:
                 mountPath: /secrets/cloudsql-instance-credentials
                 readOnly: true
           - name: cloudsql-proxy
-            image: gcr.io/cloudsql-docker/gce-proxy:1.14
+            image: gcr.io/cloudsql-docker/gce-proxy:1.19.1
             command:
```
Comment (jasiu001) [inline resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml:102]: for images > 1.14 there is no bash anymore. The current command will fail. 
please replace:
```yaml
command:
  - /bin/sh
args:
  - -c
  - "trap 'exit 0' SIGINT; /cloud_sql_proxy -instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432 -credential_file=/secrets/cloudsql-instance-credentials/credentials.json"
```
to:
```yaml
command: [ "/cloud_sql_proxy",
                     "-instances={{ .Values.global.database.managedGCP.instanceConnectionName }}=tcp:5432",
                     "-credential_file=/secrets/cloudsql-instance-credentials/credentials.json" ]
```

it was used to kill `cloud_sql` container to finish cron job (without this, one of the containers still live and cron job will never end)

File: resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml [hunk 2]
```diff
@@ -105,7 +105,7 @@ spec:
 
             {{- if eq .Values.global.database.embedded.enabled false }}
             - name: cloudsql-proxy
-              image: gcr.io/cloudsql-docker/gce-proxy:1.14
+              image: gcr.io/cloudsql-docker/gce-proxy:1.19.1
               command:
```
Comment (jasiu001) [inline resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml:109]: the same as for `environments-cleanup-job` job

File: resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml [hunk 3]
```diff
@@ -232,7 +232,7 @@ spec:
...
```
Comment (jasiu001) [inline resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml:236]: the same as for `environments-cleanup-job` job

**Review by jasiu001 — CHANGES_REQUESTED** [review]

File: resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml (hunk 1 above)
Comment (franpog859) [reply resources/kcp/charts/kyma-environment-broker/templates/environments-cleanup-job.yaml]: Ok! Done

File: resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml (hunk 2 above)
Comment (franpog859) [reply resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml]: Ok! Done

File: resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml (hunk 3 above)
Comment (franpog859) [reply resources/kcp/charts/kyma-environment-broker/templates/subaccount-cleanup-job.yaml]: Ok! Done

**Review by jasiu001 — APPROVED** [review]

**Review by jasiu001 — DISMISSED** [review]

**Comment by jasiu001** [comment]
inside `migrator-job.yaml` there are two jobs, please add "istio annotation" to both of them

**Review by jasiu001 — APPROVED** [review]

**Review by Maladie — DISMISSED** [review]

**Review by jasiu001 — APPROVED** [review]

**Review by Maladie — APPROVED** [review]

**Pull request merged**